p2p/host/autorelay/autorelay_test.go

```diff
@@ -400,11 +401,18 @@ func TestMaxAge(t *testing.T) {
 	require.Eventually(t, func() bool { return numRelays(h) > 0 }, 3*time.Second, 100*time.Millisecond)
 	relays := usedRelays(h)
 	require.Len(t, relays, 1)
+
+	waitFor := 500 * time.Millisecond
+	tick := 100 * time.Millisecond
```

Comment:
Nit: this doesn't need to be a variable. (I don't feel strongly about this, feel free to merge)